versions/3.0.4.md

```diff
@@ -2592,7 +2593,7 @@ description: Pets operations
 
 #### Reference Object
 
-A simple object to allow referencing other components in the OpenAPI document, internally and externally.
+A simple object to allow referencing other components in the OpenAPI Description, internally and externally.
```

Comment:
This suggests that "in the OAD" means "in the OAD, including documents that are only part of the OAD because of this Reference Object", which would mean that `operationRef` above, with similar wording, can also pull in otherwise-unconnected documents.  Is this how this wording is intended?

Reply:
I'm not sure what you mean by "otherwise-unconnected documents" -- I think $ref is a thing that connects documents. But perhaps to directly answer your question -- I think it is intended for $ref to be able to reference _any_ document, whether or not it was somehow "connected" through some other means.

Reply:
Yes, `$ref` has to be able to do that.  I'm pointing out that I find that the language suggests to me that the document has to somehow _already_ be part of the OAD, but the only way we define to connect things is `$ref`/`operationRef`/`mapping`.  For `$ref` it's (to us) self-evident that there's no other way, but for `operationRef` and `mapping` this wording could be read to say that you can only point `operationRef` and `mapping` to a document that has already been `$ref`'d.  So this comment is really more about `operationRef` and `mapping`.